Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.webpushd.sb.in

```diff
@@ -47,6 +47,7 @@
     (subpath "/usr/lib")
     (subpath "/System/Library/Frameworks")
     (subpath "/System/Library/PrivateFrameworks"))
+    (subpath "/System/Library/SubFrameworks"))
```

Comment:
I believe you need to remove the last closing parenthesis on the line `(subpath "/System/Library/PrivateFrameworks"))`.

Reply:
Yes my build just caught that hah